internal/tools/tool/base_test.go

```diff
@@ -0,0 +1,87 @@
+package tool
+
+import (
+	"os"
+	"testing"
+)
+
+func TestName(t *testing.T) {
+	base := Base{name: "test_tool"}
+	if base.Name() != "test_tool" {
+		t.Errorf("expected 'test_tool', but got %v", base.Name())
+	}
```

Comment:
why not use asserts?

Reply:
Replaced all use of t.Errorf, there was a lot!

Entered:

[https://github.com/microsoft/go-sqlcmd/issues/214](https://github.com/microsoft/go-sqlcmd/issues/214)

To ensure we don't regress the new clean state.